validators_instructions.md

```diff
@@ -0,0 +1,139 @@
+
+Additionally, Steward houses the Orchestrator, which is responsible for interactions with both chains via the Gravity Bridge.
+
+### **Cellars**
+
+Cellars are smart contracts that are designed to manage funds based on a particular investment strategy.
+
+### **Data Providers**
+
+Data Providers (DPs) are users or organizations who provide recommended rebalance arguments based on price or other market data to the Sommelier validators in exchange for a fee. There can be multiple Data Providers per strategy, and Validators are responsible for deciding which one they pay in exchange for data.
+
+### **Sommelier Chain**
+
+The Sommelier chain is a Cosmos SDK application chain. It is the core of Sommelier that makes decisions about whether to act on DP recommendations and executes Cellar functions. Transactions and function calls are only settled on-chain when the Validators reach consensus. In order to fulfill a strategy, Validators need a way to receive recommendation data from the Data Providers, reach consensus on whether to act on that data, and then call functions on the Cellars with the provided data as parameters. This is where Steward comes in.
+
+### **Steward**
+
+Steward runs on every Validator in the Sommelier Validator set. It runs a server to which Data Providers send recommendations as soon as market data becomes available. This payload contains identifying information about the Cellar which Steward is meant to act on, and the recommended arguments for its rebalance function parameters. When Steward receives a new recommendation, it submits a vote to rebalance the appropriate Cellar based on that data. If Validators reach a quorum, the process of rebalancing begins.
+
+Since Sommelier is a Cosmos chain, in order to provide strategies on Ethereum it needs a way to bridge assets and logic calls. Steward is responsible for running the Orchestrator, which handles relaying Cosmos transactions to Ethereum, and co-processing Ethereum transactions on Sommelier. Steward runs the Orchestrator so that Sommelier can manage Cellars on Ethereum.
+
+## Setting Up Steward
+
+In this section, let’s explore setting up steward for validators. First, ensure the sommelier chain is running. Next, create a `toml` file in the root of the application which will hold your configuration. To get a template for your configuration file, run the command below:
+
+```rust
+steward -- print-config
+```
+
+Replace the default keys in the template displayed in your terminal with your configuration. Now you have the configurations set up, let’s go through stewards commands.
+
+## Start Cosmos Mode
+
+To start the Allocation signer mode, ensure that the server section in your config file is set properly as shown below. The `address`, `client_ca_cert_path` and `port` are optional fields.
+
+```rust
+keystore = "/tmp/keystore"
+
+[server]
+address = "127.0.0.1"           // This is an optional feild.
```

Comment:
For optional fields, if we're going to include example values, we should use the defaults the application uses. I believe for `address` it will be `0.0.0.0` and for `port` it will be `5734`. I'm not certain whether optional fields should have the empty string or not be present at all (in the latter case maybe comment them out instead) but it's worth verifying.

Reply:
@cbrit I think you gave me server default address as 127.0.0.1 by mistake

Reply:
fixed

Reply:
I think the default address may have been something that was changed between the generation of this and now, so it's not unexpected.